_chapters/chapter_03.md

```diff
@@ -415,33 +430,49 @@ end
 {% endlist %}
 {% endsteps %}
 
+{% aside %}
+### Embedded RuBy
+
+You may have heard of or used HTML at some point. One of the really cool tools that Ruby has is the ability to use something called Embedded RuBy or ERB for short. 
+
+You'll notice that some of the HTML brackets in our views are slightly different now. Anything between `<%` and `%>` is actually this templating language.  
+
+Also notice that your view's file name is `index.html.erb` <- that erb on the end of the file name tells Ruby that the file needs to be interpreted for you. ERB can be used with any type of text file but in Rails, it's primarily used in the HTML views. 
+{% endaside %}
```

Comment:
Start off by defining ERB - "Erb is a templating language" ... then maybe simplify.